buildSrc/src/integTest/groovy/org/elasticsearch/gradle/JdkDownloadPluginFuncTest.groovy

```diff
@@ -185,8 +185,8 @@ class JdkDownloadPluginFuncTest extends AbstractGradleFuncTest {
 
         where:
```

Comment:
Should we do a clean inbetween build runs here to ensure it's actually getting "cached" or is that irrelevant since we aren't talking about task output here?

Reply:
It shouldn't matter but actually it doesn't harm here to make more explicit that this also works for _clean_ builds as these artifact transform outputs are not put into build output but into the gradle user home. 

FYI. If the input of a transform is produced by a task (e.g. tar task) the according transform output _is_ actually stored in the producing projects build dir folder